### DISCUSSION TITLE
Auth provider change

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#6588] [Q&A] [answered]
Hi, atm i use saml for auth. this works, but there are quirks in saml.
So we want to use oidc. Most of our users auth via saml. Is it possible to switch to oidc and all settings are the same, permission etc. ?

**Comment by si458** [comment, marked as answer]
no, because the userids would be completely different
`user//~saml:460ab1700f856xxxxxxxxxxxxxxxxxxxx`
`user//~oidc:460ab1700f856xxxxxxxxxxxxxxxxxxxx`

EDIT: you would have to edit the database manually and replace any reference of
`user//~saml:460ab1700f856xxxxxxxxxxxxxxxxxxxx`
with
`user//~oidc:460ab1700f856xxxxxxxxxxxxxxxxxxxx`

**Comment by simpsonetti** [comment]
ah, oh, okay, maybe i get this. the new oidc should be setup in feb 25.
Thanks
